app/views/items/index.html.erb

```diff
@@ -8,10 +8,26 @@
   </div>
 <% end %>
 
+<p>- - - - - - - - - -</p>
+<div id="top-5-items-by-revenue">
+  <h3>Top 5 Most Popular Items</h3>
+  <% count = 1 %>
+  <% @merchant.items.top_five_items_by_revenue.each do |item| %>
+    <div id="top-item-<%= item.name %>">
+      <%= count %>. <%= item.name %><br>
+
+      <%= link_to "#{item.name}", merchant_item_path(@merchant.id, item.id) %>
+
+      <% count +=1 %>
```

Comment:
Creative solution!  Instead of `count`, you could consider using an ordered list with the `<ol>` and `<li>` tags.